components/tag/TagActionButton.vue

```diff
@@ -17,6 +17,9 @@ const toggleFollowTag = async () => {
   else
     await client.v1.tags.follow(tag.name)
 
+  // eslint-disable-next-line vue/no-mutating-props
+  tag.following = !tag.following
```

Comment:
I think this should be before the request - using optimistic ui as we do elsewhere

Reply:
You overlooked my explanation!
https://github.com/elk-zone/elk/pull/1391#discussion_r1083544459

Reply:
I did indeed!
I think we should use try/catch then and revert the change if it throws

Reply:
okay will do it